pkg/controller/masteruserrecord/masteruserrecord_controller.go

```diff
@@ -183,15 +181,20 @@ func (r *ReconcileMasterUserRecord) ensureUserAccount(logger logr.Logger, murAcc
 		recordSpecUserAcc: murAccount,
 		logger:            logger,
 	}
-	if err := sync.synchronizeSpec(); err != nil {
-		return r.wrapErrorWithStatusUpdate(logger, mur, r.setStatusFailed(toolchainv1alpha1.MasterUserRecordUnableToSynchronizeUserAccountSpecReason), err,
-			"update of the UserAccount.spec in the cluster '%s' failed", murAccount.TargetCluster)
+	if synchronized, err := sync.synchronizeSpec(); err != nil {
+		// if we got an error while synchronizing the specs, then we probably can't update it here neither.
```

Comment:
Not necessarily. Maybe there is a temp problem with connecting the member cluster. If we get an error then we should make sure we try to wrap it in the mur status. Either here or in mur controller after calling `ensureUserAccount()` and let's be consistent. Because currently some errors are wrapped in the status inside `ensureUserAccount()` and some not. So I would probably wrap all of them in this method or return them (with error/message wrapping) to the controller and wrap them into status there.

Reply:
actually, I reverted that change, but we may not be able to update the MUR status in all cases, though.